.github/workflows/validate-windows-binaries.yml

```diff
@@ -0,0 +1,80 @@
+name: Validate binary images
+
+on:
+  push:
+    branches:
+      main
+    paths:
+      - .github/workflows/validate-windows-binaries.yml
+  pull_request:
+    paths:
+      - .github/workflows/validate-windows-binaries.yml
+jobs:
+  generate-conda-matrix:
+    uses: pytorch/test-infra/.github/workflows/generate_binary_build_matrix.yml@main
+    with:
+      package-type: conda
+      os: windows
+      channel: nightly
+  generate-wheel-matrix:
+    uses: pytorch/test-infra/.github/workflows/generate_binary_build_matrix.yml@main
+    with:
+      package-type: wheel
+      os: windows
+      channel: nightly
+  generate-libtorch-matrix:
+    uses: pytorch/test-infra/.github/workflows/generate_binary_build_matrix.yml@main
+    with:
+      package-type: libtorch
+      os: windows
+      channel: nightly
+  validate-windows-binaries-conda:
+    needs: generate-conda-matrix
+    strategy:
+      matrix:
+        ${{ fromJson(needs.generate-conda-matrix.outputs.matrix) }}
+      fail-fast: false
+    runs-on: ${{ matrix.validation_runner }}
+    steps:
+      - name: Checkout PyTorch builder
+        uses: actions/checkout@v2
+      - name: Validate binary conda
+        uses: ./.github/actions/validate-windows-binary
+        with:
+          gpu_arch_type: ${{ matrix.gpu_arch_type }}
+          gpu_arch_ver: ${{ matrix.gpu_arch_version }}
+          installation: ${{ matrix.installation }}
+          python_version: ${{ matrix.python_version }}
+  validate-windows-binaries-wheel:
+    needs: generate-wheel-matrix
+    strategy:
+      matrix:
+        ${{ fromJson(needs.generate-wheel-matrix.outputs.matrix) }}
+      fail-fast: false
+    runs-on: ${{ matrix.validation_runner }}
+    steps:
+      - name: Checkout PyTorch builder
+        uses: actions/checkout@v2
+      - name: Validate binary wheel
+        uses: ./.github/actions/validate-windows-binary
+        with:
+          gpu_arch_type: ${{ matrix.gpu_arch_type }}
+          gpu_arch_ver: ${{ matrix.gpu_arch_version }}
+          installation: ${{ matrix.installation }}
+          python_version: ${{ matrix.python_version }}
+  validate-linux-libtorch-binaries:
+    needs: generate-libtorch-matrix
+    strategy:
+      matrix:
+        ${{ fromJson(needs.generate-libtorch-matrix.outputs.matrix) }}
+      fail-fast: false
+    runs-on: "windows-2019"
+    env:
+      PYTHON_VERSION: ${{ matrix.python_version }}
+    steps:
+      - name: Install pytorch and smoke test
+        shell: powershell
+        run: |
+          $install = '${{ matrix.installation }}'
```

Comment:
Nit: I think $install can be inlined as:
```
Invoke-WebRequest -Uri ${{ matrix.installation }} -OutFile 'libtorch.zip'
```

Reply:
Yes, I think this should work:
`
Invoke-WebRequest -Uri '${{ matrix.installation }}' -OutFile 'libtorch.zip'
`